index.html

```diff
@@ -49,9 +49,9 @@ <h1>Hello, this is our content!</h1>
       </div>
       <div class="footer-heading footer-1">
         <h5><b>CS@Mines Club Sites</b></h5>
-        <a href="#"><h6>Club 1</h6></a>
-        <a href="#"><h6>Club 2</h6></a>
-        <a href="#"><h6>Club 3<h6></a>
+        <a href="https://lug.mines.edu/" target="blank"><h6>Linux Users Group</h6></a>
+        <a href="https://oresec.mines.edu/" target="blank"><h6>Mines OreSec</h6></a>
+        <a href="https://mines.campuslabs.com/engage/organization/association-for-computing-machinery-women" target="blank"><h6>ACM-W Engage<h6></a>
```

Comment:
This may look like it's working to you by opening in a new tab, but actually you're opening it in a browsing context with the name "blank". If you don't already have a browsing context with that name, it _looks_ like you're just opening it in a new tab because the browser makes a new tab with that named browsing context. But you can see what it's _actually_ doing by clicking on more than one link. It'll load the first thing you click in a new tab. Then, when you click on the next, it will replace the context of that newly opened tab with that second URL, instead of opening another  new tab.

What you probably meant to do was use the special target value `_blank`. Note the underscore (<kbd>_</kbd>).

Reply:
That's exactly what I meant to do! Thanks for the feedback :)